src/Memory.js

```diff
@@ -122,7 +126,7 @@ define([
 				index[this.getIdentity(data[i])] = i;
 			}
 
-			this.emit(eventType, event);
+			this._emit(eventType, event);
```

Comment:
This won't fly when converted since `_emit` is `private` (not `protected`), but I suspect that we might end up not even needing `_emit` separately anyway based on my comments in `Store.ts`.